src/go/README.md

```diff
@@ -9,7 +9,7 @@ Goals:
 2. Code can easily be shared with natual package paths that match file paths.
    No replace directives necessary in go.mod.
 3. Make it easy to find shared libs/patterns for use across magma targets.
-4. Other projects can easily import github.com/magma/magma code for use in
+4. Other projects can easily import github.com/magma/magma/src/go code for use in
    derivative works.
 
```

Comment:
perhaps a good place to add a few notes on:

- why isn't go.mod at $MAGMA_ROOT (package names will collide with existing dirs)
- future/todo: look at a canonical golang import path of magmacore.org/magma